e2e/test/scenarios/metrics/reproductions/metrics-reproductions.cy.spec.ts

```diff
@@ -57,89 +57,6 @@ describe("issue 47058", () => {
   });
 });
 
-describe("issue 44171", () => {
```

Comment:
That's unrelated to visualizer and was highlighted by the "add series" feature. Not sure how to replicate this using the visualizer.